app/public/js/home.js

```diff
@@ -60,11 +60,53 @@ function initMap()
 		google.maps.event.trigger(map, "resize");
 		map.setCenter(center); 
 	});
+	google.maps.event.addListener(map, 'click', function(event) {
+    locationInfo(map, event.latLng);
+  	});
 
 	conditionalProcessAddress();
 
 	$('#address').bind('keyup change', delayedProcessAddress);	
 }
+//To calculate the radian value of a point
+var rad = function(x) {
+  return x * Math.PI / 180;
+};
+// To calculate the distance between two points on a map 
+var calculateDistance = function(pt1, pt2) {
```

Comment:
Try to be consistent with the existing code.  

Replace how you define your functions with this format:
function <name>(parameters...)

For example,  function calculateDistance(pt1, pt2) {